configs/textual_inversion_sdxl_gnome_1x24gb_example.yaml

```diff
@@ -16,7 +16,7 @@ optimizer:
     optimizer_type: AdamW
 
 data_loader:
-  type: TEXTUAL_INVERSION_SD_DATA_LOADER
+  type: TEXTUAL_INVERSION_SDXL_DATA_LOADER
```

Comment:
All Stable Diffusion data loaders have been consolidated (i.e. SD 1 and SDXL now use the same dataloader), so `TEXTUAL_INVERSION_SD_DATA_LOADER` should be correct. Was this causing problems for you?